components/google_drive/actions/create-file/create-file.js

```diff
@@ -0,0 +1,244 @@
+const googleDrive = require("../../google_drive.app");
+const fs = require("fs");
+const got = require("got");
+
+module.exports = {
+  key: "google_drive-create-file",
+  name: "Create a New File",
+  description: "Create a new file from a URL or /tmp/filepath.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleDrive,
+    drive: {
+      propDefinition: [
+        googleDrive,
+        "watchedDrive",
+      ],
+    },
+    uploadType: {
+      type: "string",
+      label: "Upload Type",
+      description: `The type of upload request to the /upload URI. If you are uploading data (using an /upload URI), this field is required. If you are creating a metadata-only file, this field is not required. 
+        media - Simple upload. Upload the media only, without any metadata.
+        multipart - Multipart upload. Upload both the media and its metadata, in a single request.
+        resumable - Resumable upload. Upload the file in a resumable fashion, using a series of at least two requests where the first request includes the metadata.
+      `,
+      options: [
+        "media",
+        "multipart",
+        "resumable",
+      ],
```

Comment:
**-Minor-**
We can probably make this description shorter so that the UI has no trouble rendering it.
Maybe something like this?
```suggestion
      `,
      options: [
        {
          label: "Media (upload the media only, without any metadata)
          value: "media"
        },
        {
          label: "Multipart (upload both the media and its metadata, in a single request)
          value: "multipart"
        },
        {
          label: "Resumable (Upload the file in a resumable fashion, using a series of requests)
          value: "resumable"
        },
      ],
```

Reply:
Updated